### PR TITLE
Adding CREATE TABLE statements for DB initialization + README provided

### PR DESCRIPTION
**Pull request by christinajiyunlee**
DB is initialized in AWS RDS VPC DB instance using these CREATE TABLE statements. High level setup and data overview is provided in the README file.

### CODE REVIEW

File: db/README.md
Comment (nicolasabousawan) [inline db/README.md:4]: Can we add database connection information to the read me? We will need that information to make requests to the database from the "server"

File: db/attachment_create.sql
Comment (nicolasabousawan) [inline db/attachment_create.sql:3]: Is this the revert script? Revert scripts should be a separate file in a separate folder. I don't think this file set up will work because we would need to comment out the create script and then uncomment the drop script and then deploy. Ideally the files are ready immediately and all we need to do is all execute the files from the CLI. So can we split them and place the drop scripts in a /revert folder?

File: db/README.md
Comment (gurleen-sanghera) [inline db/README.md:34]: I guess we haven't really discussed the lengths of these different attributes, but in practice it might make a difference to have a contact number of 50 characters vs 10 for validation purposes. maybe something to make a note of.

LGTM overall :)

Comment (nicolasabousawan) [reply db/README.md:34]: Canada and the US have only 10 character numbers. Other regions, will use more (or maybe less)  
It'll be on the implementation of the backend logic (and the front end) to sanitize and validate the data. So I think 50 characters is fair

Comment (nicolasabousawan) [reply db/README.md:34]: But also thinking about it... we might want to make it just numerical and then we format the number in a lambda function instead of accepting "(" or "-" 
Its up to you, Christina!

File: db/README.md
Comment (nicolasabousawan) [reply db/README.md:4]: @kvptkr brought up a good point, we shouldn't display this info publicly... let's set it as environment variable through the lambda GUI. Let's discuss this offline or at stand up... but this is not a blocker for this PR.

**Comment by nicolasabousawan** [comment]
Comments I left are not merge blockers, but just points to consider before merging. I will let @gurleen-sanghera make the final approval on the PR just in case there are subsequent commits. Please direct message me if you have any questions as I will AFK.

**Review by gurleen-sanghera — APPROVED** [review]
Looks good!

Nicolas made some good points. Nothing is blocking this from being merged at this point, but all stuff to look into :)

File: db/attachment_create.sql
Comment (christinajiyunlee) [reply db/attachment_create.sql:3]: After consideration, since these CREATE TABLE statements are executed externally via a CLI, we determined separation of the revert scripts are not necessary for our set up.

File: db/README.md
Comment (christinajiyunlee) [reply db/README.md:34]: Yes, the 50 character logic was due to varying phone number across the world. I was thinking the formatting can be guided via the interface (like have field for area code, then phone number, etc) or be cleaned up in the actual load table functions.

**Pull request merged**
